src/Console/Report/FixReport/ReportSummary.php

```diff
@@ -50,7 +52,8 @@ public function __construct(
         int $memory,
         bool $addAppliedFixers,
         bool $isDryRun,
-        bool $isDecoratedOutput
+        bool $isDecoratedOutput,
+        bool $printFileSummary = true
```

Comment:
Will be ever use that default?

Reply:
It's used in tests, because I've changed only initialisation in `FixCommand`. But I can provide explicit value everywhere and remove default one.

Now I see that `ReportSummaryTest` should be updated too, I missed it.

Reply:
back in the past we were avoiding using default values for parameters, as we prefer explicit over implicit